packages/core/src/Badge/Badge.tsx

```diff
@@ -12,7 +12,11 @@ export { staticClasses as badgeClasses };
 
 export type HvBadgeClasses = ExtractNames<typeof useClasses>;
 
+export type HvBadgeVariant = "primary" | "subtle";
+
 export interface HvBadgeProps extends HvBaseProps {
+  /** The badge variant. */
+  variant?: HvBadgeVariant;
```

Comment:
if we're doing this to change the colors, then we should use the `color` prop like in other components, and add proper support in the component
```suggestion
  color?: HvColorAny;
```

Reply:
Updated to use `color` instead of `variant`.